Add 2.19 metrics and telemetry content (#1052)
run-ai/docs: docs/developer/metrics/metrics-api.md

# Metrics and telemetry

> ## telemetry

Reviewer: ```suggestion
## Telemetry
```

> telemetry are numeric measurements recorded at current time that are emitted from the Run:ai cluster.

Reviewer: ```suggestion
Telemetry is a numeric measurement recorded in real-time when emitted from the Run:ai cluster.
```

## Metrics

*Metrics* are numeric measurements recorded **over time** that are emitted from the Run:ai cluster. Typical metrics involve utilization, allocation, time measurements and so on. Metrics are used in Run:ai dashboards as well as in the Run:ai administration user interface.

---

> The purpose of this document is to detail the structure and purpose of metrics emitted by Run:ai to enable customers to create custom dashboards or integrate metric data into other monitoring systems.

Reviewer: ```suggestion
The purpose of this document is to detail the structure and purpose of metrics emitted by Run:ai. This enables customers to create custom dashboards or integrate metric data into other monitoring systems.
```

> Run:ai provides metrics via the [Run:ai Control-plane API](http://../admin-rest-api/overview.md). In the past, Run:ai provided metrics information via direct access to an internal metrics store. This method is deprecated but is still documented [here](http://metrics.md).

Reviewer: ```suggestion
Run:ai provides metrics via the [Run:ai Control-plane API](http://../admin-rest-api/overview.md). Previoulsy, Run:ai provided metrics information via direct access to an internal metrics store. This method is deprecated but is still documented [here](http://metrics.md).
```
